src/scenes/CommunityPage/CommunityPageView.tsx

```diff
@@ -57,6 +58,18 @@ export default function CommunityPageView({ communityRef }: Props) {
     }
   }, [descriptionRef]);
 
+  function formatUrl(str: string) {
+    let matchesUrls = str.match(VALID_URL);
+    let formattedText = str;
+    matchesUrls?.map((url: string) => {
+      formattedText = formattedText.replace(url, `[${url}](${url})`);
+    });
+
+    return formattedText;
+  }
```

Comment:
nit: functions defined within components should be wrapped in `useCallback`.

that said, because this is more of a utility function, can we just put this in `util/formatUrl.ts`?

Reply:
Sure! Just moved it to `util` folder